protocols/nogaim.h

```diff
@@ -265,6 +265,12 @@ struct prpl {
 	/* If null, equivalent to handle_cmp( ic->acc->user, who ) */
 	gboolean (* handle_is_self) (struct im_connection *, const char *who);
 
+	/* This is called when the user uses the /kick IRC command.
+	 * - 'who' is a handle to kick
+	 * - 'message' is a kick message or NULL
+	 */
+	void (* chat_kick)	(struct groupchat *, char *who, char *message);
```

Comment:
I guess you're putting it here for slightly better binary compatibility of plugins? Otherwise, it's better sorted around chat_invite().